packages/geo/src/lib/layer/layer-legend-list/layer-legend-list.component.html

```diff
@@ -1,35 +1,37 @@
 
-<mat-slide-toggle 
-tooltip-position="above"
-matTooltipShowDelay="500"
-[matTooltip]="'igo.geo.layer.legend.showAll' | translate"
-[checked]="showAllLegendsValue" class="mat-typography" 
-*ngIf="(layersInUi$ | async).length && allowShowAllLegends" [labelPosition]="'before'" (change)="toggleShowAllLegends($event.checked)">
-  {{'igo.geo.layer.legend.showAll' | translate}}
-</mat-slide-toggle>
-<mat-divider *ngIf="(layersInUi$ | async).length && allowShowAllLegends"></mat-divider>
-<igo-list [navigation]="false" [selection]="false">
-  <ng-template ngFor let-layer let-i="index" [ngForOf]="layers$ | async">
-    <igo-layer-legend-item *ngIf="!(excludeBaseLayers && layer.baseLayer)"
-        igoListItem [layer]="layer"
-        [updateLegendOnResolutionChange]="updateLegendOnResolutionChange">
-    </igo-layer-legend-item>
-  </ng-template>
-</igo-list>
+<div style="max-height: 90%; overflow: auto;">
```

Comment:
Maybe add a css class instead of inline css.

Reply:
done